pkg/types/stackerfile.go

```diff
@@ -220,8 +220,10 @@ func NewStackerfile(stackerfile string, validateHash bool, substitutions []strin
 	for _, name := range sf.FileOrder {
 		layer := sf.internal[name]
 		if layer.WasLegacyImport() {
-			log.Warnf("Deprecated 'import' directive used in layer '%s' of file '%s'. "+
-				"Change from 'import' to 'imports', and '/stacker/' to '/stacker/imports/'", name, stackerfile)
+			log.Warnf("'import' directive used in layer '%s' inside file '%s' is deprecated. "+
```

Comment:
You dropped the `Change from 'import' -> 'imports'` which I think is critical to allowing folks to fix things ASAP without loading up a webpage.  I'd keep the original line 223/224, and then add your new lines.

Reply:
The message occurs every time a layer is loaded. its kind of annoying.
I figure the url is a place to document how to fix it.  

> "Change from 'import' to 'imports', and '/stacker/' to '/stacker/imports/'"

probably wasn't completely clear to anyone that wasn't familiar anyway.

If people think we should put that back in, i'm fine to do so... its just that the message is so long.

Reply:
For something we are deprecating, best to complain?

Reply:
done.